docs/advanced.md

```diff
@@ -0,0 +1,71 @@
+# Advanced Options
+
+{% hint style="warning" %}
+This is a section detailing the more advanced parts of using Northstar, with some of them having a chance to potentially mess with your game. Setting your level too high, for example, will cause you to need to reset your level back to 1.
+{% endhint %}
+
+## Mod profiles <a href="#profiles" id="profiles"></a>
```

Comment:
Might be worth making _Profiles_ it's own sections so like

```
## Mod profiles

### Vanilla profile

### Regular profile
```

and actually maybe even switch regular and vanilla around as vanilla is a "special case" of a regular mod profile.